soh/src/overlays/actors/ovl_Boss_Tw/z_boss_tw.c

```diff
@@ -5012,11 +5059,15 @@ void BossTw_DrawEffects(GlobalContext* globalCtx) {
                 sp18F++;
                 BossTw_InitRand(1, 0x71AC, 0x263A);
             }
+            
+            FrameInterpolation_RecordCloseChild();
```

Comment:
Move this to the end of the for loop and remove the following inner `j` open/close.